backend/src/activitypub/routes/webfinger.spec.js

```diff
@@ -0,0 +1,117 @@
+import { handler } from './webfinger'
+import Factory from '../../seed/factories'
+import { getDriver } from '../../bootstrap/neo4j'
+
+let resource
+let res
+let json
+let status
+let contentType
```

Comment:
Is there any reason you prefer multi-line `let` statements over one-liners?
Just a question...

Reply:
No, I think a linter with a previous configuration complained about it.